app/views/cloud_subnet/new.html.haml

```diff
@@ -0,0 +1,105 @@
+      .col-md-8
+        = select_tag("ems_id",
+                      options_for_select(@network_provider_choices.sort),
+                      "ng-model"                    => "cloudSubnetModel.ems_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet Name')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "name",
+                            'ng-model'     => "cloudSubnetModel.name",
+                            'ng-maxlength' => 128,
+                            :miqrequired   => true,
+                            :checkchange   => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('Network')
+      .col-md-8
+        = select_tag("network_id",
+                      options_for_select(@network_choices.sort),
+                      "ng-model"                    => "cloudSubnetModel.network_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet CIDR')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "cidr",
+                            'ng-model'     => "cloudSubnetModel.cidr",
+                            'ng-maxlength' => 20,
+                            :miqrequired   => false,
+                            :checkchange   => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('Subnet Gateway')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                            :name          => "gateway",
+                            'ng-model'     => "cloudSubnetModel.gateway",
+                            'ng-maxlength' => 20,
+                            :miqrequired   => false,
+                            :checkchange   => true}
+  %h3
+    = _('Placement')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Cloud Tenant')
+      .col-md-8
+        = select_tag("cloud_tenant_id",
+                      options_for_select(@cloud_tenant_choices.sort),
+                      "ng-model"                    => "cloudSubnetModel.cloud_tenant_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+
+  %table{:width => '100%'}
```

Comment:
@himdel, actually that doesn't work for the new view. I get a 'save' button triggering 'update', instead of, I suppose should be a 'add' to trigger 'new'.